solutions/usaco-1107.mdx

```diff
@@ -66,5 +66,83 @@ print(abs(ans))
 ```
 
 </PySection>
+	
+<JavaSection>
+	
+```java
+import java.io.*;
+import java.util.*;
+
+public class YearOfTheCow {
+	static String[] animals = {"Ox", "Tiger", "Rabbit", "Dragon", "Snake", 
+	"Horse", "Goat", "Monkey", "Rooster", "Dog", "Pig", "Rat"};
+
+	static Map<String, Integer> when_born = new TreeMap<>();
+
+	// given a year, return what animal the year is
+	static String getAnimal(int year)
+	{
+		int a = 0, y = 2021;
```

Comment:
Better to spell out the variables to currAnimal and maybe currYear or something like that